### ISSUE TITLE
ItyFuzz Extremely Slow For Kyber Elastic Pools

### ISSUE THREAD

**Issue by g0rk** [fuzzland/ityfuzz#7]
I achieved a rate of 40000 exec/s with other contracts, but when it comes to Kyber Elastic Pool, the performance is really poor, only 500 exec/s.

Here is an example: https://bscscan.com/address/0xd7422179457700d7d15c111f4466e50e6eb36ec1

Can you fix this?

**Comment by g0rk** [comment]
Also tried local build with "flashloan_v2" enabled in Cargo.toml.

Command to reproduce: 
```
./target/release/cli -o -t 0xd7422179457700d7d15c111f4466e50e6eb36ec1 -c BSC --onchain-block-number 0 -f -i --onchain-local-proxy-addr http://localhost:5003
```

Without proxy it is even slower (100 exec/s)
```
./target/release/cli -o -t 0xd7422179457700d7d15c111f4466e50e6eb36ec1 -c BSC --onchain-block-number 0 -f -i 
```

**Comment by shouc** [comment]
fixed in #12
